blocks/editable/block.scss

```diff
@@ -0,0 +1,16 @@
+.drop-cap-true .blocks-editable__tinymce:not( :focus )  {
```

Comment:
> Yeah, we can probably move this and remove the tinymce stuff. I though it was somewhat odd that the drop cap was in editable and not in text. I had to do an ack to find where the drop cap was. I think the idea behind it though is that there could be other fields that want a drop cap when using editable?

In the backend, if the dropcap CSS is applied to the editable, deleting characters behaves really weirdly. I can't recall exactly what went wrong, but we solved it by deciding that the dropcap is applied only in the backend when your cursor isn't actually on the editable block. We'd obv. want it there always on the frontend.